client/blocks/authentication/social/social-tos.jsx

```diff
@@ -3,8 +3,39 @@ import { localize } from 'i18n-calypso';
 import { connect } from 'react-redux';
 import { isWooOAuth2Client } from 'calypso/lib/oauth2-clients';
 import { getCurrentOAuth2Client } from 'calypso/state/oauth2-clients/ui/selectors';
+import isWooCommerceCoreProfilerFlow from 'calypso/state/selectors/is-woocommerce-core-profiler-flow';
 
 function SocialAuthToS( props ) {
+	if ( props.isWooCoreProfilerFlow ) {
+		return (
+			<p className="auth-form__social-buttons-tos">
+				{ props.translate(
+					'If you continue with Google or Apple, you agree to our' +
```

Comment:
I am not sure if we should concatenate strings like this. Can we make it in one line?

Reply:
There shouldn't be any problem in this type of concatenation, as we only have one `props.translate`, it's more of a readibility thing (that was already there) 😄